src/test/javascript/spec/component/modeling-editor/modeling-editor.component.spec.ts

```diff
@@ -225,4 +226,37 @@ describe('ModelingEditorComponent', () => {
         const spanText = statusHint.query(By.css('span'))?.nativeElement?.textContent;
         expect(spanText).toBe('Saving...');
     });
+
+    it('should handle explanation input change', () => {
+        fixture.detectChanges();
+        const spy = jest.spyOn(fixture.componentInstance.explanationChange, 'emit');
+
+        const newExplanation = 'New Explanation';
+        fixture.componentInstance.onExplanationInput(newExplanation);
+
+        expect(spy).toHaveBeenCalledOnce();
```

Comment:
```suggestion
        expect(spy).toHaveBeenCalledTimes(1);
```
That's just what we usually use afaik